wagtail/snippets/templates/wagtailsnippets/snippets/revisions/_actions.html

```diff
@@ -3,11 +3,13 @@
 <td {% if column.classname %}class="{{ column.classname }}"{% endif %}>
     {% if revision_enabled and instance.revision %}
         {% with revision=instance.revision latest_revision=object.get_latest_revision previous_revision=instance.revision.get_previous %}
-            <span>{{ value }}</span>
-            {% if draftstate_enabled %}
-                {% if instance.action == 'wagtail.publish' and revision == object.live_revision %}{% trans 'Live version' as status_label %}{% status status_label classname="w-status--primary" %}
-                {% elif instance.content_changed and revision == latest_revision %}{% trans 'Current draft' as status_label %}{% status status_label classname="w-status--primary" %}{% endif %}
-            {% endif %}
+            <div class="w-flex w-items-center w-gap-2">
```

Comment:
It's probably time we consolidate this with page history's `_actions.html` template, but ideally the page history view should also be refactored to use the generic history view + tables framework that will be done in #10914.